cmd/release-controller/sync_verify.go

```diff
@@ -80,6 +90,12 @@ func (c *Controller) ensureVerificationJobs(release *Release, releaseTag *imagev
 			jobLabels := map[string]string{
 				"release.openshift.io/verify": "true",
 			}
+			if analysisVerification {
+				err := c.launchAnalysisJobs(release, jobName, verifyType, releaseTag, previousTag, previousReleasePullSpec)
```

Comment:
Can we move this down into `ensureProwJobForReleaseTag` so that we keep all the prowjob creation in one spot?

Reply:
There is a little bit of trickery here...
The creation of all the `analysis` jobs (`c.launchAnalysisJobs(...)`) is actually calling `ensureProwJobForReleaseTag()`, under the hood, multiple times with a slightly modified `verifyType`: [here](https://github.com/openshift/release-controller/blob/148959822b46660b05b5f403580dba56333054ba/cmd/release-controller/sync_analysis.go#L23-L38)

The next call to `ensureProwJobForReleaseTag` is  the actual creation of the `aggregator` itself. 

If you're OK with it, I'd prefer to leave this as is?